pdns/pdns_recursor.cc

```diff
@@ -3147,6 +3146,8 @@ static int serviceMain(int argc, char*argv[])
 
   openssl_thread_setup();
   openssl_seed();
+  /* setup rng before chroot */
+  (void)dns_random(1);
```

Comment:
This seems like an ugly way to initialize the PRNG, perhaps we could make a version of `dns_random_init()` taking no parameter instead?